lib/widgets/dialogs/app_info_dialog.dart

```diff
@@ -95,16 +77,25 @@ class _AppInfoDialogState extends State<AppInfoDialog> {
 											child: Icon(Icons.keyboard_arrow_right, color: Colors.grey)
 										),
 										leading: Icon(Icons.filter_vintage),
-										onTap: () => _openBrowserpage(context, githubURL)
+										onTap: () => ExternalURL.github.openBrowserPage(context)
 									),
 									ListTile(
-										title: Text(AppLocales.of(context).translate('$_settingsKey.terms')),
+										title: Text(AppLocales.of(context).translate('$_settingsKey.termsOfUse')),
 										trailing: Padding(
 											padding: EdgeInsets.only(left: 4.0, top: 2.0),
 											child: Icon(Icons.keyboard_arrow_right, color: Colors.grey)
 										),
 										leading: Icon(Icons.description),
-										onTap: () => _openBrowserpage(context, termsURL)
+										onTap: () => ExternalURL.termsOfUse.openBrowserPage(context)
```

Comment:
One not so great thing about our header - it takes all the screen on mobile. You don't even know there is anything further down. We should at least have the page title visible at all times
![image](https://user-images.githubusercontent.com/8643919/104108600-5cde3200-52c6-11eb-8e91-9ed701431e72.png)

Reply:
True, not very user friendly as of now.

Reply:
You could get rid of some stuff for now: there is no pl language, github button is not necessary on the sub-pages..

Reply:
Yep, that's what I have in mind too.